src/pod/pod_pubkey.rs

```diff
@@ -0,0 +1,18 @@
+use bytemuck::{Pod, Zeroable};
+use solana_program::pubkey::Pubkey;
+
+use crate::pod::Nullable;
+
+#[repr(C)]
+#[derive(Copy, Clone, Default, Pod, Zeroable, PartialEq, Eq, Hash, Debug)]
+pub struct OptionalPubkey(pub Pubkey);
+
+impl Nullable for OptionalPubkey {
+    fn is_some(&self) -> bool {
+        self.0 != Pubkey::default()
+    }
+
+    fn is_none(&self) -> bool {
+        self.0 == Pubkey::default()
```

Comment:
I guess it might be just an inversion of `is_some`, but it's nit-picky.

Reply:
I normally like that but TBH I don't trust the Rust compiler to do it efficiently.